views/layout.html

```diff
@@ -3,7 +3,7 @@
     {{> partials-head}}
   {{/head}}
   {{$pageTitle}}
-    {{$header}}{{/header}} &ndash; GOV.UK
+    {{#errorlist.length}}Error: {{/errorlist.length}}{{$header}}{{/header}} &ndash; GOV.UK
```

Comment:
Might be worth adding a translation string like:
```suggestion
    {{#errorlist.length}}{{#t}}errors.title.prefix{{/t}} {{/errorlist.length}}{{$header}}{{/header}} &ndash; GOV.UK
```
(and then setting it in the right place within `translations/src`)

Reply:
Ah pants, only just seen this - I've done something similar which will hopefully do the job - what do you recon @zuzak ?